### PR TITLE
Python modules not intended for execution should not be marked as scripts

### PR DESCRIPTION
**Pull request by yarikoptic**
didn't test...

### CODE REVIEW

File: libexec/python/docker/Makefile.am [hunk 1]
```diff
@@ -1,6 +1,6 @@
 scriptlibexecdir = $(libexecdir)/singularity/python/docker
 
-dist_scriptlibexec_SCRIPTS = api.py __init__.py converter.py
+dist_scriptlibexec_DATA = api.py __init__.py converter.py
```
Comment (vsoch) [inline libexec/python/docker/Makefile.am:4]: @gmkurtzer does this fit with your standard for the Makefiles? What is the distinction betwen DATA and SCRIPTS?

**Comment by yarikoptic** [comment]
On November 28, 2016 6:51:01 PM EST, Vanessa Sochat <notifications@github.com> wrote:
>vsoch commented on this pull request.
>
>
>
>> @@ -1,6 +1,6 @@
> scriptlibexecdir = $(libexecdir)/singularity/python/docker
> 
>-dist_scriptlibexec_SCRIPTS = api.py __init__.py converter.py
>+dist_scriptlibexec_DATA = api.py __init__.py converter.py
> 
>
>@gmkurtzer does this fit with your standard for the Makefiles? What is
>the distinction betwen DATA and SCRIPTS?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/singularityware/singularity/pull/351#pullrequestreview-10442025

This is autotools setup - scripts is something you execute, so they acquire executable permissions. data if something not so special ;-)

**Comment by vsoch** [comment]
I think I just heard a little whimper or squeak of sadness from the python folder...

*...I'm special too!*

**Comment by yarikoptic** [comment]
I might say more -- it is too special ;-)

**Comment by vsoch** [comment]
"*eeee!*"

**Comment by vsoch** [comment]
I think this should be good then, although @gmkurtzer will need to put in final say to do the merge. @yarikoptic would you mind rebasing with master to update the PR?

File: libexec/python/docker/Makefile.am (hunk 1 above)
Comment (gmkurtzer) [reply libexec/python/docker/Makefile.am:4]: You nailed it. SCRIPTS are marked exec, while DATA isn't.

**Comment by gmkurtzer** [comment]
Looks good to me! Thanks! :D

**Comment by vsoch** [comment]
Awesome, thanks everyone!

**Pull request merged**